.github/workflows/buf-ci.yaml

```diff
@@ -0,0 +1,25 @@
+name: Buf CI
```

Comment:
Move to `buf-ci.yaml` to avoid `buf.yaml`

Reply:
Most commits don't touch the protos, so the old config had a path filter. I see the new action recommends the following config for this case:
```
on:
  push:
    paths:
      - '**.proto'
      - '**/buf.yaml'
      - '**/buf.lock'
      - '**/buf.md'
      - '**/README.md'
      - '**/LICENSE'
```
We probably also want to add `.github/workflows/buf-ci.yaml` to that list.

Reply:
Added.